docs/content.zh/release-notes/flink-1.13.md

```diff
@@ -0,0 +1,248 @@
+
+##### [FLINK-20752](https://issues.apache.org/jira/browse/FLINK-20752)
+
+The Failure Rate Restart Strategy was allowing 1 less restart per interval as configured. Users
+wishing to keep the current behavior should reduce the maximum number of allowed failures per
+interval by 1.
+
+#### Unaligned checkpoint recovery may lead to corrupted data stream
+
+##### [FLINK-20654](https://issues.apache.org/jira/browse/FLINK-20654)
+
+Using unaligned checkpoints in Flink 1.12.0 combined with two/multiple inputs tasks or with union
+inputs for single input tasks can result in corrupted state.
+
+This can happen if a new checkpoint is triggered before recovery is fully completed. For state to be
+corrupted a task with two or more input gates must receive a checkpoint barrier exactly at the same
+time this task finishes recovering spilled in-flight data. In such case this new checkpoint can
+succeed, with corrupted/missing in-flight data, which will result in various
+deserialisation/corrupted data stream errors when someone attempts to recover from such corrupted
+checkpoint.
+
+Using unaligned checkpoints in Flink 1.12.1, a corruption may occur in the checkpoint following a
+declined checkpoint.
+
+A late barrier of a canceled checkpoint may lead to buffers being not written into the successive
+checkpoint, such that recovery is not possible. This happens, when the next checkpoint barrier
+arrives at a given operator before all previous barriers arrived, which can only happen after
+cancellation in unaligned checkpoints.
+
+#### Support rescaling for Unaligned Checkpoints
+
+##### [FLINK-17979](https://issues.apache.org/jira/browse/FLINK-17979)
+
+While recovering from unaligned checkpoints, users can now change the parallelism of the job. This
+change allows users to quickly upscale the job under backpressure.
+
+### SQL
+
+#### Officially deprecate the legacy planner
+
+##### [FLINK-21709](https://issues.apache.org/jira/browse/FLINK-21709)
+
+The old planner of the Table & SQL API is deprecated and will be dropped in Flink 1.14. This means
+that both the BatchTableEnvironment and DataSet API interop are reaching end of life. Use the
+unified TableEnvironment for batch and stream processing with the new planner, or the DataStream API
+in batch execution mode.
+
+#### Disable problematic cast conversion between NUMERIC type and TIMESTAMP type
+
+##### [FLINK-21698](https://issues.apache.org/jira/browse/FLINK-21698)
+
+The CAST operation between `NUMERIC` type and `TIMESTAMP` type is problematic and is disabled now,
+e.g.
+`CAST(numeric AS TIMESTAMP(3))` is disabled and should use `TO_TIMESTAMP(FROM_UNIXTIME(numeric))`
+instead.
+
+#### Support 'USE MODULES' syntax
+
+##### [FLINK-21298](https://issues.apache.org/jira/browse/FLINK-21298)
+
+The term MODULES is a reserved keyword now. Use backticks to escape column names and other
+identifiers with this name.
+
+#### Update `TableResult.collect()/TableResult.print()` to the new type system
```

Comment:
The usage of inline code blocks seems a bit inconsistent, e.g. here it is put into inline code, but in most other places it isn't.